frontend/src/components/RecordsList.tsx

```diff
@@ -227,26 +226,26 @@ const RecordsList: FunctionComponent<IRecordsListProps> = ({
       )}
 
       <Toolbar
-        searchTerm={searchTerm}
-        input={inputVal}
-        setInput={setInputVal}
+        dataName={dataName}
+        userSearchVal={userSearchVal}
+        setUserSearchVal={setUserSearchVal}
         handleSearch={handleSearch}
-        clearInput={() => {
-          setCustomFilterVals && setCustomFilterVals([]);
-          setSearchVal([]);
+        clearUserSearchVal={() => {
+          setCustomSearchVals && setCustomSearchVals([]);
+          setParsedSearchVals([]);
         }}
         matchingResultsCount={`${remoteCount?.toLocaleString()} matching ${
-          remoteCount > 1 ? searchTerm : searchTerm.slice(0, -1)
+          remoteCount > 1 ? dataName : dataName.slice(0, -1)
         }`}
         handleDownload={handleDownload}
-        customUI={customFilterUI}
+        customUI={customToolbarUI}
       />
 
       <AutoSizer>
         {({ width }) => (
           <div
-            className="ag-theme-alpine"
-            style={{ height: 540, width: width }}
+            className={`ag-theme-alpine ${styles.tableHeight}`}
+            style={{ width: width }}
```

Comment:
This is part of a minor change that sets the AG Grid table height to fill up the available space, removing the unused gap below the table.